drivers/filesystems/ext2/src/fileinfo.c

```diff
@@ -137,6 +137,7 @@ Ext2QueryFileInformation (IN PEXT2_IRP_CONTEXT IrpContext)
             }
 
             FileBasicInformation = (PFILE_BASIC_INFORMATION) Buffer;
+            RtlZeroBytes(FileBasicInformation, sizeof(FILE_BASIC_INFORMATION));
```

Comment:
The `Buffer` is zeroed out already. Why zeroing for the second time is needed? If the error was triggered here, maybe we have a case where `Length` and `sizeof(FILE_BASIC_INFORMATION)` don't match?
Or `RtlZeroMemory(Buffer, Length)` just got optimized out?

Reply:
I removed that redundant function.